README.md

```diff
@@ -398,7 +398,7 @@ services:
           - peer1.org1.hf.chainhero.io
 ```
 
-**Be careful, you need to change something in the file !**
+## **Be careful, you need to change something in the file !**
```

Comment:
Reduce of one level here and remove bold: "### Be careful, you need to change something in the file !"